situp-core/src/main/resources/situp-default.yml

```diff
@@ -2,15 +2,17 @@ otel-es-raw-pipeline:
   source:
     apm_trace_source:
   sink:
-    pipeline:
-      name: "otel-es-enrich-pipeline"
+    - pipeline:
+        name: "otel-es-enrich-pipeline"
   workers: 5
   delay: 3000
 otel-es-enrich-pipeline:
   source:
     pipeline:
       name: "otel-es-raw-pipeline"
   sink:
-    elasticsearch:
-      hosts: ["https://search-sample-app-test-lqwynrnd2ikcuzfsrdilv4stbq.us-west-2.es.amazonaws.com"]
+    - elasticsearch:
+        hosts: ["https://search-sample-app-test-lqwynrnd2ikcuzfsrdilv4stbq.us-west-2.es.amazonaws.com"]
```

Comment:
This really should not be in the default yaml.

Reply:
Why should situp have a default yaml? This doesn't make sense, we should probable avoid this and ask users to always provide the yml fils as an argument.

Reply:
I agree with you but would prefer it have that change in its separate PR.